documentation/inventory_file_v2.md

```diff
@@ -129,8 +131,6 @@ groups:
     config: 
       ssh: 
         password: password
-    facts: 
```

Comment:
I think the may actually just need to be unindented, so it's at the same level as `groups` and `version`, based on the "top-level" in the next paragraph. Maybe "There is a fact on the 'all' group" would be better phrasing? @hestonhoffman  would know

Reply:
Yeah agreed - that `fact` should be at the top level. Let's replace that sentence with "There is a second fact, `operatingsystem: CentOS`, in the `all` group."

Reply:
That seems like it would no longer exemplify that "Inventory files are not context-aware." in the following paragraph.